.buildkite/fpm-pipeline.yml

```diff
@@ -6,16 +6,14 @@ env:
   DOCKER_REGISTRY: "docker.elastic.co"
   STAGING_IMAGE: "${DOCKER_REGISTRY}/observability-ci"
   MAKEFILE: "fpm"
-  CHANGESET_FILE: ".buildkite/scripts/changeset/fpm"
-  DOCKER_FILTER_REF: "docker.elastic.co/beats-dev"
   BUILDX: "0"
 
 steps:
   - label: ":linux: multiarch Linux x86_64/arm64 FPM docker image"
-    key: "build-and-publish-ubuntu-x86"
+    key: "build-and-publish-ubuntu-x86-fpm"
```

Comment:
avoid clashing with the existing keys, use the format `build...` so the `pre-command` hook keeps working.